### PR TITLE
Include OrgRole participations in organizations and persons participation listing

### LINKED ISSUE #2151: Include OrgRole particpations in contact detail views
The participation listing in the contact detail views (person and organization) only shows direct participations but not the one where a OrgRole ist participating. 

bildschirmfoto_2016-09-12_um_15_16_52

### PR DESCRIPTION
**Pull request by phgross**
Closes #2151

### CODE REVIEW

**Comment by phgross** [comment]
@deiferni

File: opengever/contact/models/participation.py [hunk 1]
```diff
@@ -20,6 +22,26 @@ class ParticipationQuery(BaseQuery):
     def by_dossier(self, dossier):
         return self.filter_by(dossier_oguid=Oguid.for_object(dossier))
 
+    def by_organization(self, organization):
+        """Returns both direct ContactParticapations and OrgRoleParticipation
```
Comment (deiferni) [inline opengever/contact/models/participation.py]: ~~direct~~

File: opengever/contact/models/participation.py [hunk 2]
```diff
@@ -20,6 +22,26 @@ class ParticipationQuery(BaseQuery):
...
+        of the given organization.
+        """
+        return self._org_role_join().filter(
+            or_(OrgRole.organization == organization,
+                ContactParticipation.contact == organization))
+
+    def by_person(self, person):
+        """Returns both direct ContactParticapations and OrgRoleParticipation
```
Comment (deiferni) [inline opengever/contact/models/participation.py]: ~~direct~~

**Comment by deiferni** [comment]
👍

**Pull request merged**